data-model/fdo-type/annotation/0.3.0/schema/annotation.json

```diff
@@ -264,54 +271,13 @@
     "dcterms:creator": {
       "type": "object",
       "description": "Object containing information on who created the object",
-      "properties": {
-        "@id": {
-          "type": "string",
-          "description": "The PID if the type is SoftwareAgent, the ORCID if the type is Person and the ROR if the type is Organisation.",
-          "examples": [
-            "https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX",
-            "https://doi.org/10.123/XXX-XXX-XXX",
-            "https://ror.org/0566bfb96",
-            "https://orcid.org/0000-0002-5669-2769"
-          ],
-          "pattern": "https://(doi\\.org/10\\.\\d+/[-a-zA-Z0-9]+$|hdl\\.handle\\.net/[a-zA-Z0-9.]+/[-a-zA-Z0-9]+$|ror\\.org/0[a-z|0-9]{6}[0-9]{2}$|orcid\\.org/\\d{4}-\\d{4}-\\d{4}-\\d{3}[\\dX]+)$"
-        },
-        "@type": {
-          "type": "string",
-          "description": "The type of the creator.",
-          "enum": [
-            "foaf:Person",
-            "foaf:Organisation",
-            "as:Application"
-          ]
-        },
-        "foaf:name": {
-          "type": "string",
-          "description": "The name of the creator. http://xmlns.com/foaf/0.1/#term_name",
-          "examples": [
-            "Gbif linker service"
-          ]
-        }
-      },
-      "required": [
-        "@id",
-        "@type"
-      ],
-      "additionalProperties": false
+      "$ref": "https://schemas.dissco.tech/schemas/fdo-type/shared-model/0.3.0/agent.json"
```

Comment:
yes. I think is better to have the agent as a separate object.

Reply:
:+1: